solidity/contracts/libraries/operator/Groups.sol

```diff
@@ -85,7 +85,7 @@ library Groups {
         bytes memory groupPubKey,
         address[] memory members,
         bytes memory misbehaved
-    ) internal {
+    ) public {
```

Comment:
From the peanut gallery, this change looks odd without some sort of ACL. Anyway, flagging it for proper review

Reply:
That change is inherited from #1851 and should be reviewed there; I wish there was a better way to chain PRs than to make the latter PRs merge into the earlier ones.

Reply:
Making that PR the base branch for this one would help a bit

Reply:
Probably a faux pas but I flipped it out. Much easier diff IMO